swift-extensions/ColorExtensions.swift

```diff
@@ -1,5 +1,5 @@
 import UIKit
-import TRIKOT_FRAMEWORK_NAME
```

Comment:
I feel like this will break our `trikot.patron` boilerplate code replacement.

Reply:
actually the reason i changed them manually is because the code replacement didn't work, if i don't commit this change the build will fail locally